### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by AsmahanM**
I've added an event listener so that a new list is created when the button is clicked. 
Next step it to append the list items!

Let me know if you spot any error or think any changes can be made.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+//select the tv button element.
+//create eventListener to initiate function when tv button is clicked.
```
Comment (PJSalter) [inline index.js:17]: Really enjoy reading the clear pseudocode 🤩  really helpful explanations ✨

File: index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    //create inner div for input and button element.
+    const inputDiv = document.createElement('div')
+    inputDiv.classList.add("list-input")
+    
+    newList.appendChild(inputDiv)
+    
+
+    //create input element.
+    const inputTitle = document.createElement('input')
```
Comment (PJSalter) [inline index.js:36]: very descriptive name for the variable ⭐

File: index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+//function to create a new list onclick.
+function createList() {
+
+    //create outer div to contain list.
+    const newList = document.createElement('div')
+    newList.classList.add("list-container")
+   
+
+    //create inner div for input and button element.
```
Comment (mariaalouisaa) [inline index.js:28]: Great comments and spacing, makes the function nice and easy to read!

File: index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    inputBtn.classList.add("list-input")
+
+    inputDiv.appendChild(inputBtn)
+    
+
+    //append the new list to tv button.
+    const element = document.querySelector('.main')
+    element.appendChild(newList)
+
+}
```
Comment (mariaalouisaa) [inline index.js:57]: Function look amazing!

File: style.css [hunk 5]
```diff
@@ -1,17 +1,6 @@
...
-Yellow(#EFD30B)
-Green(#73ba25)
-Red(#C92A2E)
-Light Grey(#BFBFBF)
-Dark Grey (#474747)
- */
 
 body {
   margin: 0;
-  /* font-family: ? */
```
Comment (mariaalouisaa) [inline style.css:45]: Thanks for tidying up 🙂

**Review by mariaalouisaa — APPROVED** [review]
All looks great!

File: index.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+//select the tv button element.
+//create eventListener to initiate function when tv button is clicked.
+document.querySelector("#new-list").addEventListener("click", createList)
+
+//function to create a new list onclick.
+function createList() {
+
+    //create outer div to contain list.
```
Comment (PJSalter) [inline index.js:23]: very neat code 😄 

Thank you for adding this JavaScript code for creating a new list ✨

File: index.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    const newList = document.createElement('div')
+    newList.classList.add("list-container")
+   
+
+    //create inner div for input and button element.
+    const inputDiv = document.createElement('div')
+    inputDiv.classList.add("list-input")
```
Comment (PJSalter) [inline index.js:30]: I see so the classList.add() adds a new one every time the user types?

This is a great way at addressing the task 🙌 🐰 ✨

File: index.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    
+    newList.appendChild(inputDiv)
```
Comment (PJSalter) [inline index.js:32]: Thank You this is a brilliant way of using appendChild 😺

File: index.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    inputTitle.placeholder="Add new title."
+    inputTitle.classList.add("list-input")
+   
+    inputDiv.appendChild(inputTitle)
+
+
+    //create button to submit input.
+    const inputBtn = document.createElement('button')
+    inputBtn.id="add-title"
+    inputBtn.innerHTML="Add"
```
Comment (PJSalter) [inline index.js:47]: So once the User clicks the button then does it respond immediately with a new list item ? as many as user wants?

🤩 Interesting way in creating this, great work ✨

File: style.css [hunk 10]
```diff
@@ -62,6 +51,83 @@ h1 {
...
+}
+
+.list-input button{
+  position: relative;
+  float: right;
+  font-weight: 500;
+  font-size: 16px;
+  background-color: black;
+  border: none;
+  color: #FF00FF;
```
Comment (PJSalter) [inline style.css:185]: the CSS Variables can be used here for the colour choice in keeping them consistent

File: style.css [hunk 11]
```diff
@@ -62,6 +51,83 @@ h1 {
...
+ font-family:'Gochi Hand', 'Just Another Hand', cursive;
+ font-size: 16px;
+ border-style: solid;
+ border-color: #FF00FF;
+ background-color: black;
+ border-width: 2px;
+ letter-spacing: 0.1em;
+}
+
+.list-input input:focus{
```
Comment (PJSalter) [inline style.css:174]: Great use in focus selector to pick out the element in that focus area 

great work ⭐

File: style.css [hunk 12]
```diff
@@ -62,6 +51,83 @@ h1 {
...
+
+}
+
+.list-input input{
+ width: 75%;
+ height: 30px;
+ padding: 10px;
+ color: #FF00FF;
+ position: relative;
+ border-radius: 0px;
```
Comment (PJSalter) [inline style.css:164]: just a query but do we need the border-radius at 0px?

File: style.css [hunk 13]
```diff
@@ -62,6 +51,83 @@ h1 {
   z-index: 10;
 }
 
+/* To-Do List Skeleton: */
+.list-container {
+  background: black;
+  width: 40%;
+  top: 30%;
```
Comment (PJSalter) [inline style.css:139]: Great use in using the % for sizing 😄 ✨

File: style.css [hunk 14]
```diff
@@ -62,6 +51,83 @@ h1 {
...
+  left: 30%;
+  right: 50%;
+  min-height: 100px;
+  min-width: 300px;
+  border-radius: 0px;
+  z-index: 2;
```
Comment (PJSalter) [inline style.css:145]: Great use of z-index 🙌

File: style.css [hunk 15]
```diff
@@ -1,17 +1,6 @@
-/* COLOUR HEX CODES:
```
Comment (PJSalter) [inline style.css:1]: Thank you for removing the code thats not being used 😺

**Review by PJSalter — APPROVED** [review]

**Pull request merged**